src/Data/These.purs

```diff
@@ -119,6 +124,8 @@ maybeThese = case _, _ of
   Just a, Just b -> Just (Both a b)
   Nothing, Nothing -> Nothing
 
+-- | Values stored in a `Both` are rewrapped as a `Tuple`. Otherwise,
+-- | the missing value is filled by this function's corresponding argument.
```

Comment:
Is there a way to refer to `a` and `b` as 'default' values? That's typical language for similar functions like `fromMaybe`.

Reply:
I updated the language after checking the `fromMaybe` rendering.